srml/contract/src/lib.rs

```diff
@@ -14,41 +14,64 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate. If not, see <http://www.gnu.org/licenses/>.
 
-//! Smart-contract module for runtime; Allows deployment and execution of smart-contracts
-//! expressed in WebAssembly.
+//! # Contract Module
+//! 
+//! The contract module provides functionality to deploy and execute smart-contracts expressed in WebAssembly.
+//! To use it in your module, you need to implement the contract [`Trait`].
```

Comment:
That's true, but it is not the full picture. 

Technically we need to add this to the `construct_runtime!` macro ([here](https://github.com/paritytech/substrate/blob/aee743eb5d9d8698dbc8d58f848e5cc15638af37/node/runtime/src/lib.rs#L215) is the example)

Reply:
Ah  I see thanks! 

rephrasing to: 

The contract module provides functionality for the runtime to deploy and execute smart-contracts expressed in WebAssembly. To add this functionality, you need to add the contract module to `construct_runtime!` in the runtime module.